.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: publish
+
+on:
+  release:
+    types: [published]
+  workflow_run:
```

Comment:
Good for testing but we'll want to remove this before merge, yes? Or am I misunderstanding?